lms/static/scripts/frontend_apps/components/ErrorModal.tsx

```diff
@@ -78,6 +92,7 @@ export default function ErrorModal({
           {cancelLabel}
         </Button>
       )}
+      {extraActions}
```

Comment:
`extraActions` here is rendered, when present, after any cancel button, when present. Are there situations in which we could end up with:

`<cancel button> [Edit assignment] <retry button>`

or

`<cancel button> [Edit assignment]`

? Perhaps `extraActions` could come first? Also perhaps you have already thought through these cases...

Reply:
For the current use case in `LaunchErrorDialog`, there is no Cancel option, so the issue doesn't arise. I am not sure which will make the most sense for future use cases. When "Edit assignment" had a more button-like appearance it seemed sensible to put it next to the primary action. With the revised link-like appearance, that makes less sense. I'll move it to the left of "Cancel" for now, but I expect we might need to revisit in future.